sql/src/test/java/io/crate/operation/tablefunctions/GenerateSeriesFunctionTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.operation.tablefunctions;
+
+import org.junit.Test;
+
+public class GenerateSeriesFunctionTest extends AbstractTableFunctionsTest {
```

Comment:
the assertions in this tests are generated on the assumptions of the actual implementation, i would prefer using literal strings or other means of testing by expecting results that are reviewed and not generated with an alrogrithm which is actually a copy of the implementation